saleor/graphql/shipping/types.py

```diff
@@ -56,11 +56,17 @@
 class ShippingMethodChannelListing(
     ModelObjectType[models.ShippingMethodChannelListing]
 ):
-    id = graphene.GlobalID(required=True)
-    channel = graphene.Field(Channel, required=True)
-    maximum_order_price = graphene.Field(Money)
-    minimum_order_price = graphene.Field(Money)
-    price = graphene.Field(Money)
+    id = graphene.GlobalID(
+        required=True, description="The ID of shipping method channel listing."
+    )
+    channel = graphene.Field(
+        Channel,
+        required=True,
+        description="The channel associated with the shipping method channel listing.",
+    )
+    maximum_order_price = graphene.Field(Money, description="Maximum order price.")
+    minimum_order_price = graphene.Field(Money, description="Minimum order price.")
+    price = graphene.Field(Money, description="Shipping price.")
```

Comment:
Maybe let's make it more descriptive - "Price of the shipping method in the associated channel".